internal/db/migrations/2024-05-22.0-ingest_payments.sql

```diff
@@ -20,7 +20,7 @@ CREATE TABLE ingest_payments (
   dest_asset_issuer text NOT NULL,
   dest_amount bigint NOT NULL,
   created_at timestamp with time zone NOT NULL,
-  memo text,
+  memo text NULL,
```

Comment:
Now that we've got a deployed version, we should be careful about modifying the migrations.

Reply:
For sure! But in this case it should be safe, since this is just making the default value explicit.

Reply:
That's great! 🚀